client/src/components/Breadcrumb/Breadcrumb.scss

```diff
@@ -79,11 +79,16 @@
   font-weight: normal;
   line-height: 24px;
   @include text-truncate;
+  display: flex;
+  align-items: center;
 }
 
 .breadcrumb__icon::before {
   margin-left: 1rem;
   font-size: 1.5rem;
   cursor: pointer;
-  vertical-align: middle;
+}
+
+.breadcrumb__icon {
+  margin-top: 5px;
 }
```

Comment:
This doesn't seem to be helping with vertical alignment:

<img width="168" alt="image" src="https://user-images.githubusercontent.com/242621/82267341-27e04080-99c0-11ea-89a5-433d5c0d780c.png">

Reply:
Bug in js, wasn't adding the class to the icon